topics/imaging/tutorials/detection-of-mitoflashes/tutorial.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: tutorial_hands_on
+
+title: Tracking of mitochondria and capturing mitoflashes
+zenodo_link: https://zenodo.org/records/14071552
+questions:
+- What are mitoflashes, and why are they relevant for understanding mitochondrial function?
+- How can bioinformatics and image analysis tools help in tracking and analyzing mitochondrial dynamics?
+objectives:
+- Understand the biological significance of mitoflashes and their implications for cellular health.
+- Learn to track mitochondrial movements in live-cell imaging data.
+- Use image analysis tools to detect and quantify mitoflashes.
+time_estimation: 1H
+key_points:
+- Mitoflashes are superoxide events in mitochondria that signal functional states and stress responses.
+- Bioinformatics tools can facilitate the visualization, tracking, and quantitative analysis of these events.
+- Proper tracking and analysis of mitoflashes contribute to understanding mitochondrial behavior and health in cells.
+
+contributions:
+  authorship:
+    - dianichj
+    - kostrykin
+tags:
+  - bioimaging
+  - mitoflash
+  - mitochondria
+    
+---
+
+# Introduction
```

Comment:
@bgruening if you're reviewing things in the future, could we trouble you to please ask folks to remove the '# introduction' section header, they should just start the text, rather than including this header.

Reply:
Ok, I will try to remember. It does not look that bad :)

I fixed it in Leonids PR.